c_src/refobjects.h

```diff
@@ -73,16 +73,15 @@ class ErlRefObject : public RefObject
     //  owns the shutdown (Erlang or async C)
     void * volatile * m_ErlangThisPtr;
 
-    // 1 once InitiateCloseRequest starts,
-    // 2 other pointers to "this" released
-    // 3 final RefDec and destructor executing
-    volatile uint32_t m_CloseRequested;
-
     leveldb::port::Mutex   m_CloseMutex;        //!< for condition wait
     leveldb::port::CondVar m_CloseCond;         //!< for notification of user's finish
 
 protected:
```

Comment:
You introduced the GetCloseRequested() method to ensure that m_CloseRequested is read cleanly, so I recommend making m_CloseRequested private and making the few direct references to m_CloseRequested (in DbObject) use calls to GetClosedRequested() instead.